docs/rst/user_manual/participants/xml.rst

```diff
@@ -0,0 +1,63 @@
+.. include:: ../../exports/alias.include
+
+.. _user_manual_participants_xml:
+
+###############
+XML Participant
+###############
+
+This type of :term:`Participant` refers to a :term:`DomainParticipant` that uses QoS profiles loaded from XML files to be configured.
+
+|fastdds| supports XML to fully configure a DomainParticipant.
+Using XML configuration, user have whole access to the full configuration of a DDS DomainParticipant.
+Check following `documentation <https://fast-dds.docs.eprosima.com/en/latest/fastdds/xml_configuration/domainparticipant.html>` for further information about configuring a DDS DomainParticipant with XML.
+For further information regarding how to load XML configuration files to |ddsrouter| check following section :ref:`user_manual_configuration_load_xml`.
+
+.. note::
+
+    This kind of Participant is only meant for expert users.
+    Creating an XML configuration is not an easy task, and there is a high probability that messing with internal configurations of DDS makes DDS Router to not work as expected.
+
+
+Use case
+========
+
+Use this Participant to fully configure a DomainParticipant, its discovery methods, transport options, DDS QoS, etc.
+Main use case for this Participant is using **DDS Security**, that requires XML configuration from user side.
+
+.. TODO
+
+    Add link to security documentation
+
+
+Kind aliases
+============
+
+* ``xml``
+* ``XML``
+
+
+Configuration
+=============
+
+XML Participant allows configure a profile name.
+Such name will be used as QoS profile when creating the internal DomainParticipant.
+Whole DomainParticipant configuration settings must be configured via XML, |ddsrouter| will not configure any attribute or QoS for it (that is why this participant is not advisable for non expert users).
+
+* To configure the profile, check the following section
+  :ref:`Configuration section <user_manual_configuration_profile>`.
+
+
+Configuration Example
+=====================
+
+Configure a XML Participant that uses Initial Peers to communicate via WAN using TCP.
+It listens for clients in public IP ``82.0.0.1`` in port ``11600`` in ``TCP``.
+
+.. code-block:: yaml
+
```

Comment:
Add this to the example

```yaml
xml:                                                              # 1
  files:
    - "./xml_configuration.xml"                                   # 2
  raw: |                                                          # 3
    <?xml version="1.0" encoding="UTF-8" ?>
    <profiles xmlns="http://www.eprosima.com/XMLSchemas/fastRTPS_Profiles" >
        <participant profile_name="custom_participant_configuration">
            <domainId>1</domainId>
            <rtps></rtps>
        </participant>
    </profiles>
```

Reply:
This is the example of the Participant. The xml part is described in configuration, and it is linked before